src/getCurrentPosition/getCurrentPosition.web.tsx

```diff
@@ -0,0 +1,15 @@
+type GetCurrentPositionOptions = {
+    maximumAge: number;
+    timeout: number;
+    enableHighAccuracy: boolean;
+};
+
+const getCurrentPosition = (success: (response: GeolocationPosition) => void, error: (error: GeolocationPositionError) => void, options: GetCurrentPositionOptions) => {
+    if (!('geolocation' in navigator)) {
```

Comment:
Is `navigator` a global provided by the library? I don't know where this comes from 🤔

Reply:
shouldn't navigator capital case `Navigator` instead of `navigator`?

Reply:
It's actually navigator uncapitalized https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator